tests/unittests/benchmark/test_benchmark_client.py

```diff
@@ -164,46 +129,54 @@ def test_create_with_only_name(self):
 
             assert "Benchmark {} does not exist in DB".format(name) in str(exc.value)
 
-    def test_create_with_invalid_algorithms(self):
+    def test_create_with_different_configure(self, benchmark_config_py):
```

Comment:
The logging should be captured and tested with caplog to make sure the warning was logged.